pytest_pootle/fixtures/models/user.py

```diff
@@ -12,6 +12,18 @@
 from pytest_pootle.env import TEST_USERS
 
 
+@pytest.fixture(
+    scope="session",
+    params=["nobody", "admin", "member", "member2"])
+def request_users(request):
+    from django.contrib.auth import get_user_model
+
+    info = TEST_USERS[request.param]
```

Comment:
...but only because they are session-scoped, so the object will be cached between tests, even though the db has been rolled back/updated